tests/numpy/test_creation_manipulation.py

```diff
@@ -162,6 +188,14 @@ def check_dim(_np_a, _ns_a):
     check_expand_and_squeeze(np_A, axes=(0, 5, 6))
     check_expand_and_squeeze(np_A, axes=(2, 3, 5, 6, 7))
 
+    check_swapaxes(np_A, axis1=0, axis2=1)
+    check_swapaxes(np_A, axis1=0, axis2=2)
+    check_swapaxes(np_A, axis1=0, axis2=3)
+    check_swapaxes(np_A, axis1=1, axis2=2)
+    check_swapaxes(np_A, axis1=1, axis2=3)
+    check_swapaxes(np_A, axis1=2, axis2=3)
+    check_swapaxes(np_A, axis1=3, axis2=3)
```

Comment:
Wondering, why test `axis1 == axis2 == 3` only and not 0, 1, and 2?

Reply:
There's no specific reason,  just wanted to make sure that the case in which `axis1` and `axis2` are the same works well. I just changed the code to test all the possible combinations.